index.js

```diff
@@ -3,13 +3,130 @@ const { displayWordSoFar, isGameWon, isGameLost } = require("./gamelogic");
 
 function game(word, guesses) {
   console.log("Dit heb je tot nu toe geraden: ", guesses);
+  let letter
+  do {
+    letter = question("Raad een letter: ");
+    if (letter.length > 1) {
+      console.log("je mag maar EEN letter tegelijk invoeren")
+    }
+    function isCharacterALetter(char) {
+      return (/[a-zA-Z]/).test(char)
+    }
+    if (!isCharacterALetter(letter)) {
+      console.log("Je moet een LETTER invoeren")
+    }
+  }
+  while (letter.length > 1 || !isCharacterALetter(letter))
+  letter = letter.toLowerCase()
+  word = word.toLowerCase()
```

Comment:
Is word.toLowerCase() nodig? Je letters worden al omgezet naar lowerCase, je kunt dan nooit een woord krijgen dat hoofdletters krijgt. Je hebt zelf ook geen woorden met hoofdletter in je lijstje staan en de gebruiker kan niet zelf een woord toevoegen.